docs/_tutorials/messagevpn-with-queue-java.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: tutorials
+title: Message VPN creation with Queue – Java 
+summary: Sample integration into a Java-based management tool using the SEMP Java Client Library
+icon: management-integration-java.png
+---
+
+#### Goals
+
+The goals of this tutorial are to:
+
+* Introduce the Solace Element Management Protocol (SEMP) object model for Solace message-VPNs and its resources
+* Demonstrate how to integrate the management of Solace message routers into Java-based DevOps tools using the SEMP Java Client Library
+
+This tutorial will guide you through the steps of creating a sample Java-based tool to administer basic message-VPNs and queues.
+
+Let's review the admin objectives:
+
+* **Creating message-VPNs** is a convenient way to slice a Solace message router to separated independent virtual routers, multiplying its use. There are many use cases including security, controlled share of functionality and capacity. For developers, one way to make use of a message-VPN is to share a VMR with everyone having a dedicated virtual message router for development use. Once a message-VPN is created, the client can connect to it with the assigned username and password using basic internal authentication.
```

Comment:
Can we avoid the use of "virtual message router". In the field this has proven to be confusing. It is preferred to define and use the term "message-vpn" since this matches with our UI and CLI.